observation_portal/accounts/serializers.py

```diff
@@ -78,8 +77,7 @@ def validate_email(self, data):
         return data
 
     def update(self, instance, validated_data):
-        user_update_fields = ['email', 'username', 'last_name', 'first_name']
-        instance.username = validated_data.get('username', instance.username)
+        user_update_fields = ['email', 'last_name', 'first_name']
```

Comment:
This is to stop allowing user's to update their username, since the username will be the 'fixed' field used when updating client user accounts. I think this is inline with other User management systems out there - there are other problematic issues with allowing users to change usernames since they are stored in the data files as a username.